cmd/kube-apiserver/app/server.go

```diff
@@ -80,6 +80,7 @@ type APIServer struct {
 	CorsAllowedOriginList      util.StringList
 	AllowPrivileged            bool
 	PortalNet                  util.IPNet // TODO: make this a list
+	ServiceNodePorts           util.PortRange
```

Comment:
Public implies that we definitely expose it publicly, which isn't necessarily true (e.g. if k8s is running on private network).  What we're doing it is exposing the service on a port on the minions, so we want to find a name that captured that idea (and didn't over-promise that it would be public).